include/morphio/errorMessages.h

```diff
@@ -9,35 +9,50 @@
 #include <morphio/mut/section.h>
 
 namespace morphio {
-/**
-   Set the maximum number of warnings to be printed on screen
-**/
+/** Set the maximum number of warnings to be printed on screen **/
 void set_maximum_warnings(int n_warnings);
+/** Set whether to interpet warning as errors **/
 void set_raise_warnings(bool is_raise);
+/** Set a warning to ignore **/
 void set_ignored_warning(Warning warning, bool ignore = true);
+/** Set an array of warnings to ignore **/
 void set_ignored_warning(const std::vector<Warning>& warning, bool ignore = true);
-
+/** print a warning **/
 void printError(Warning warning, const std::string& msg);
 
 namespace readers {
-enum ErrorLevel { INFO, WARNING, ERROR };
 
+/** Level of error reporting **/
+enum ErrorLevel {
+    INFO,     //!< Info
+    WARNING,  //!< Warning
+    ERROR     //!< Error
+};
+
+/** Debug info for error messages **/
 struct DebugInfo {
   public:
+    /** Constructor
+
+        \param filename morphology filename.
+     */
     DebugInfo(std::string filename = "")
         : _filename(filename) {}
 
+    /** Set line number within morphology file */
```

Comment:
where the section is defined? What does `sectionId` do here?

Reply:
```suggestion
    /** Stores line number within morphology file */
```